backend/src/parser/TheParser.ts

```diff
@@ -28,8 +28,10 @@ export default class TheParser {
     // Bump this version when introducing breaking changes to the parser.
     // Content will be re-parsed and saved on access when this version changes.
     static readonly VERSION = 2;
+    readonly domain;
 
-    constructor() {
+    constructor(domain = 'orbitar.local') {
```

Comment:
We should use prod URL by default. Otherwise in case someone will miss arguments during dev - they won't notice it in dev but it will be failing in production. It should be opposite.
Or maybe load HOST_URL from env variables

Reply:
I think a better way is to remove the default value here altogether.